src/Enhavo/Bundle/NewsletterBundle/Exception/DeliveryException.php

```diff
@@ -9,7 +9,6 @@
 namespace Enhavo\Bundle\NewsletterBundle\Exception;
 
 
-class SendException extends \Exception
+class DeliveryException extends \Exception
```

Comment:
Why we call it `DeliveryException` instead of `SendException`. Delivery sounds more unusually in a mail context and I think we can't control the delivery. Only the sending.